src/lib/cli/sonarrc-generator.ts

```diff
@@ -59,9 +59,9 @@ export const initSonarrc = async () => {
 
     const questions = [
         {
-            choices: collectorKeys,
-            message: 'What collector do you want to use?',
-            name: 'collector',
+            choices: connectorKeys,
+            message: 'What connector do you want to use?',
```

Comment:
That should be handled separately from this PR.

I've opened: https://github.com/sonarwhal/sonar/issues/361.